packages/gatsby-plugin-sharp/README.md

```diff
@@ -22,7 +22,15 @@ images. By default it uses a quality setting of [50-75].
 
 ```javascript
 // In your gatsby-config.js
-plugins: [`gatsby-plugin-sharp`]
+plugins: [
+  {
+    resolve: `gatsby-plugin-sharp`,
+    options: {
+      useMozJpeg: process.env.GATSBY_JPEG_ENCODER === `MOZJPEG`,
```

Comment:
I think you're documenting defaults here, but this seems kinda weird. It seems to be sorta contrary to the docs, because here we're using a plugin option _and_ an environment variable, when the README specifies we only need one or the other.

Reply:
I was debating whether or not to put a pure Boolean or not.... I feel that if I add a pure Boolean in the documentation, the code should be updated so that `GATSBY_JPEG_ENCODER` should always override the options supplied.

I thought that documenting it this way led to less confusion in terms of backwards compatibility and functionality.

Reply:
I was debating whether or not to put a pure Boolean or not.... I feel that if I add a pure Boolean in the documentation, the code should be updated so that `GATSBY_JPEG_ENCODER` should always override the options supplied.

I thought that documenting it this way led to less confusion in terms of backwards compatibility and functionality.